client/packages/host/src/utils.ts

```diff
@@ -1,7 +1,24 @@
-import { ReactNode } from 'react';
+import { ReactNode, useState, useEffect } from 'react';
 
 export const getBadgeProps = (value?: number) => ({
   badgeContent: (value ?? 0) as ReactNode,
   max: 99,
   color: 'primary' as 'primary' | 'default',
 });
+
+export const useDeviceDetect = () => {
+  const [isMobile, setMobile] = useState(false);
+
+  useEffect(() => {
+    const userAgent =
+      typeof window.navigator === 'undefined' ? '' : navigator.userAgent;
+    const mobile = Boolean(
+      userAgent.match(
+        /Android|BlackBerry|iPhone|iPad|iPod|Opera Mini|IEMobile|WPDesktop/i
```

Comment:
I guess this list is probably fine for our purposes, but if you wanted to, you could try one of the more foolproof ways of detecting mobile that are listed here: https://stackdiary.com/detect-mobile-browser-javascript/

(In particular, I'm looking at "TouchEvent" and "Window.matchMedia")

Probably not super important for now, but maybe add a comment with a link to that page for future reference.